src/snowflake/connector/log_configuration.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+#
+from __future__ import annotations
+
+import inspect
+import json
+import logging
+import os
+
+from snowflake.connector import SnowflakeConnection
+from snowflake.connector.secret_detector import SecretDetector
+
+
+def is_full_path(path: str) -> bool:
+    return os.path.isabs(path)
+
+
+class EasyLoggingConfigPython:
+    def __init__(self, connection_parameters=None):
+        self.log_path = None
+        self.log_level = None
+        self.connection_parameters = connection_parameters
+        self.CLIENT_CONFIG_FILE = (
+            connection_parameters["CLIENT_CONFIG_FILE"]
+            if connection_parameters and "CLIENT_CONFIG_FILE" in connection_parameters
+            else None
+        )
+        self.SF_CLIENT_CONFIG_FILE = (
+            os.environ["SF_CLIENT_CONFIG_FILE"]
+            if "SF_CLIENT_CONFIG_FILE" in os.environ
+            else None
+        )
+        self.config_file_name = "sf_client_config.json"
```

Comment:
Python connector has toml file, still we need a new file?

Reply:
Let's wait for @sfc-gh-bwarsaw  's input on this topic

Reply:
Agree with @sfc-gh-igarish - is there a requirement that the log config file must be JSON, or some other reason this can't be a section in the `~/snowflake/config.toml` file?   If there isn't then I suggest adding a `[logging]` section with `level` and `path` keys to that TOML file.  Then you wouldn't have to also add support for another environment variable because we already have env overload of the location of the TOML file.